### PR TITLE
68 convert to num

### PR DESCRIPTION
**Pull request by wgsda**
## Description
Addition for converting ranges to numerical values. It converts each response of a single choice question to the mean value of the range. For multiple choice questions, currently the sum of all numerical values from the given answers is computed into a single value. If the given range is open ended (e.g. >2500), the value is set to the lower boundary (e.g. 2500).

## Checklist

- [ ] I created an issue and discussed solutions with others.
- [ ] I wrote tests or updated tests regarding my changes.
- [ ] I follow the style guidelines of this project.
- [ ] I have commented my code. So, other can understand what I did.
- [ ] I checked if all tests passing after my changes.

## PR review

Anyone in the community is free to review the PR once the tests have passed.

- [ ] Add labels and milestones to the PR so it can be classified.
- [ ] Check that all items from the **checklist** are resolved.
- [ ] Is this pull request ready for review?

### CODE REVIEW

**Comment by fudw** [comment]
Is this ready for review? If so, I will start one because I think there are a number of changes needed before it can be merged.

**Comment by wgsda** [comment]
Yes, you can take a look on it. Please let me know the open issues.

File: n2survey/lime/survey.py
Comment (fudw) [inline n2survey/lime/survey.py:151]: Please write complete sentences as labels. You can even copy paste the original question labels because here the answers are "essentially" changing dtype nothing more.

**Review by fudw — CHANGES_REQUESTED** [review]
Nice work! A number of minor changes are needed but the overall style is consistent with the rest and the code works as intended. Should be good to merge after these small things are fixed. Great job!

File: n2survey/lime/transformations.py
Comment (lheckmann) [reply n2survey/lime/transformations.py:310]: Okay, I realized that B1b is multiple choice just in the dummy file, but for the actual survey files they managed to change it to be single choice not multiple. I will adjust the dummy files so you can see the actual situation.

Comment (wgsda) [reply n2survey/lime/transformations.py:310]: Thanks for resolving this! Since this transformation now only applies to single-choice questions, I've removed the multiple-choice part of the code.

File: tests/test_lime_structure.py
Comment (wgsda) [reply tests/test_lime_structure.py:92]: I can't remember changing anything here. If possible, this should be discarded.

File: tests/test_lime_transformations.py
Comment (wgsda) [reply tests/test_lime_transformations.py:206]: Thanks for making me aware of this. I've added tests for all questions.

File: n2survey/lime/transformations.py
Comment (wgsda) [reply n2survey/lime/transformations.py:305]: I've removed the whole function since it's not necessary anymore. Should also apply to some other review comments.

File: tests/test_lime_structure.py
Comment (wgsda) [reply tests/test_lime_structure.py]: Thanks! - Done.

**Comment by fudw** [comment]
Updated the function naming and a few docstrings myself. All tests pass and it's ready to be merged.

**Pull request merged**
